jangjuseong/app.js

```diff
@@ -194,6 +199,31 @@ app.post('/likes/:userId/:postId', async (req, res) => {
   return res.status(201).json({ message: 'likeCreated' });
 });
 
+app.get('/users/signup', async (req, res) => {
+  const { email, password } = req.body;
+
+  const [userData] = await mysqlDataSource.query(
+    `
+    SELECT * 
+    FROM users
+    WHERE email=?
+    `,
+    [email]
+  );
+
+  const payLoad = { email: userData.email };
```

Comment:
토큰을 암호화하고 복호화하여 인가 과정을 위해 필요한 payload에는 개인의 민감한 정보가 포함되어서는 안됩니다.

유저의 데이터 중 유저를 식별할 수 있으나, 무의미한 데이터는 users 테이블의 각 id 값이 될 것입니다.
유저의 id 값을 payload에 설정해주세요!
```suggestion
  const payLoad = { userId: userData.id };
```